clang/include/clang/Basic/Builtins.td

```diff
@@ -209,6 +209,18 @@ def FmaxF16F128 : Builtin, F16F128MathTemplate {
   let Prototype = "T(T, T)";
 }
 
+def MinNum : Builtin {
```

Comment:
Should the keep the f prefix convention? I suppose we aren't following a libm function, so it might make sense to drop it